build_tools/python/benchmark_suites/iree/x86_64_benchmarks.py

```diff
@@ -91,11 +80,12 @@ def _generate_experimental(
         # Disabled due to https://github.com/openxla/iree/issues/12772.
         torch_models.EFFICIENTNET_B7_FP32_TORCH,
     ]
+    models = list(map(lambda x: x[0], model_groups.x86_64_MODELS))
     gen_configs = [
         iree_definitions.ModuleGenerationConfig.build(
             compile_config=self.CASCADELAKE_FUSE_PADDING_COMPILE_CONFIG,
             imported_model=iree_definitions.ImportedModel.from_model(model))
-        for model in model_groups.SMALL + model_groups.LARGE
+        for model in models
         if model not in excluded_models_for_experiments
     ]
     # We run the fuse padding compiler config under 4 threads only.
```

Comment:
Do we want to run all these models under 4 threads? This filtering logic is starting to look complicated enough that I think it might be better to just have a list of `X86_64_MODELS_FOR_EXPERIMENTAL`

Reply:
Good point. I'll make these changes in a different PR.

Reply:
Actually, I just created one.